nextcord/application_command.py

```diff
@@ -1189,6 +1190,10 @@ async def favorite_dog(interaction: Interaction, dog: str):
             To use inputs from other options inputted in the command, you can add them as arguments to the autocomplete
             callback. The order of the arguments does not matter, but the names do.
 
+            If you are using :py:class:`nextcord.Member` or :py:class:`nextcord.user` typehints in
```

Comment:
```suggestion
            If you are using :class:`nextcord.Member` or :class:`nextcord.user` typehints in
```
Sphinx knows it is Python